CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
+## 0.9.0
+
+* Breaking change: Changed signature of `CodedBufferWriter.writeTo` to require
+  `Uint8List` for performance.
+* More Dart 2 fixes.
+
 ## 0.8.0
 
 * Breaking change: Added generics to RpcClient.invoke(). Proto files must be
   rebuilt using Dart protoc_plugin version 0.8.0 or newer to match.
 * Dart 2 fixes.
-* Breaking change: Changed signature of `CodedBufferWriter.writeTo` to require
```

Comment:
@mraleph – careful w/ versions + breaking changes!

Reply:
Ah, I misunderstood how CHANGELOG.md is structured. I thought 0.8.0 is the current development version - rather than currently released version.

Reply:
No worries. Just an FYI...